docs/developers/tutorials/staking-contract.md

```diff
@@ -271,14 +259,7 @@ If you wanted to use testnet, the proxy would be "https://testnet-gateway.multiv
 More details can be found [here](/developers/constants/).
 :::
 
-The only thing you need to edit is the USER_PEM variable with the previously created PEM file's path.
-
-To run this snippet, we're going to use the MultiversX IDE extension again. Open the extension in VSCode from the left-hand menu, right-click on the contract name, and select the `Run Contract Snippet` option. This should open a menu in at the top:  
-![img](/developers/staking-contract-tutorial-img/snippet.png)
-
-For now, we only have one option, as we only have a single function in our file, but any bash function we write in the snippets.sh file will appear there. Now, select the deploy option and let's deploy the contract.
-
-[comment]: # (mx-context-auto)
+The things you need to edit is the PEM variable and the PROJECT path with your local paths.
```

Comment:
_... the CLI parameters `--pem` and `--project`_

Reply:
done